CMakeLists.txt

```diff
@@ -192,14 +199,20 @@ endif()
 # We want Kokkos to be built with C++14, since that's what we're using in
 # Parthenon.
 set(Kokkos_CXX_STANDARD 14)
-if(DEFINED EXTERNAL_KOKKOS AND EXISTS ${EXTERNAL_KOKKOS})
-  find_package(Kokkos 3 REQUIRED PATHS "${EXTERNAL_KOKKOS}" NO_DEFAULT_PATH)
-elseif(EXISTS ${Kokkos_ROOT}/CMakeLists.txt)
-  add_subdirectory(${Kokkos_ROOT} Kokkos)
-elseif(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/external/Kokkos/CMakeLists.txt)
-  add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/external/Kokkos Kokkos)
+if (PARTHENON_EXTERNAL_KOKKOS)
+    find_package(Kokkos 3)
+    if (NOT Kokkos_FOUND)
+        unset(PARTHENON_EXTERNAL_KOKKOS CACHE)
+        message(FATAL_ERROR "Could not find external Kokkos. Consider importing a Kokkos installation into your environment or unsetting PARTHENON_EXTERNAL_KOKKOS")
```

Comment:
The error message isn't very useful, I would say "... setting to off, i.e. `-DPARTHENON_EXTERNAL_KOKKOS=OFF`.